### DISCUSSION TITLE
can't deploy service in v4

### DISCUSSION THREAD

**Discussion by dukeim** [serverless/serverless#12695] [Q&A]
Hello guys.
I am creating a new service following the steps indicated in https://www.serverless.com/framework/docs/providers/aws/guide/credentials
 I don't have an AWS Access Key and Secret configured in my windows 10, so i am created a Provider into serverless dashboard, but when i try to deploy the service i'm getting the following error:
![deploy_error](https://github.com/user-attachments/assets/f012256b-1ccd-4679-8ee6-36b0c27cae00)

This is the provider created during the service creation:
![aws_provider](https://github.com/user-attachments/assets/e3b88dcc-fb7e-47e9-a20b-45d9867d5933)
If i run the command serverless login, the login to dashboards works.
![dashboard_login](https://github.com/user-attachments/assets/b09ec4f5-393e-4dca-a847-a623f56aff05)

Then, what is missing? Why serverless framework can't authenticate using the provider created to deploy the service? 

Thanks for your help

**Comment by czubocha** [comment]
Hi @dukeim, do you have the `app` and `org` keys specified in your `serverless.yml`? These are required to enable Dashboard features for the service.
